packages/rich-list-core/src/controller/AddressParser/index.ts

```diff
@@ -0,0 +1,58 @@
+import { OP_DEFI_TX, OPCode, DfTx, toOPCodes } from '@defichain/jellyfish-transaction'
+import { ApiClient, blockchain as defid } from '@defichain/jellyfish-api-core'
+import { SmartBuffer } from 'smart-buffer'
+import { DfTxAddressParser } from './dftx/_abstract'
+import { NetworkName } from '@defichain/jellyfish-network'
+import { AccountToUtxosParser } from './dftx/AccountToUtxos'
+import { UtxoAddressParser } from './utxo'
+
```

Comment:
Technically they can be called "saga" rather than "controller" for a more accurate description of what they do.